### ISSUE TITLE
Link in README.md

### ISSUE THREAD

**Issue by albert-github** [vim/vim#7507]
**Describe the bug**
The link in the file `README.md` on the main directory:
```
![Vim Logo](https://github.com/vim/vim/blob/master/runtime/vimlogo.gif)
```
is not correct, it points to a non-image file (in fact a HTML file), but GitHub is able to correctly this information. In programs like doxygen and pandoc the above link is converted to:
```
<img src="https://github.com/vim/vim/raw/master/runtime/vimlogo.gif" alt="Vim Logo" class="inline"/>
```
and when trying to display this image this results in the alternative text "Vim Logo".

**To Reproduce**
1. take the README.md
1. create an empty Doxyfile (for doxygen)
1. run doxygen
1. view the result in a browser

**Expected behavior**
Correct link so the image is rendered properly.

**Environment (please complete the following information):**
 - Vim version: current master
 - OS: independent

**Additional context**
Possible solutions, change the link in:
```
![Vim Logo](https://github.com/vim/vim/raw/master/runtime/vimlogo.gif)
```
or
```
![Vim Logo](https://github.com/vim/vim/blob/master/runtime/vimlogo.gif?raw=true)
```
where the first solution is the preferred one.

**Comment by brammool** [comment]
Yes, it should be "raw" instead of 'blob", I'll include the fix.

**Comment by albert-github** [comment]
Unfortunately I don't see the change yet in vim 8.2.2195

**Comment by chrisbra** [comment]
change is included in 4072ba571babd733d1800c082416fe8fe3c28ed7

**Comment by brammool** [comment]
> change is included in 4072ba571babd733d1800c082416fe8fe3c28ed7

I tried making it look a bit better using a table, but it turns out the
preview isn't quite like the result on github.

Main problem is that the logo is resized.  I haven't found a way to
avoid that.

Also, the table should not show any borders.

Anyone know github-mardown well enough to fix this?

-- 
hundred-and-one symptoms of being an internet addict:
13. You refer to going to the bathroom as downloading.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
